be/src/exec/pipeline/pipeline_driver_executor.cpp

```diff
@@ -172,6 +173,11 @@ void GlobalDriverExecutor::_worker_thread() {
                 _finalize_driver(driver, runtime_state, driver_state);
                 break;
             }
+            case EPOCH_FINISH: {
+                _blocked_driver_poller->add_parked_driver(driver);
+                _epoch_finalize_driver(driver, runtime_state, driver_state);
```

Comment:
Maybe calling `_epoch_finalize_driver` before `add_parked_driver`. Otherwise, could the driver put back from the parked queue to ready queue, when `_epoch_finalize_driver` is executed?

Reply:
fixed.